src/docs/operator-guides/migrating-from-outside-sharetribe/index.md

```diff
@@ -109,12 +215,202 @@ key like this:
                :im.email/verified true}
 ```
 
+The Intermediary file validation also checks that the email address
+format is valid.
+
+- Required attributes for `:im.user`
+  - `:im.user/primaryEmail`
+  - `:im.user/role`
+  - `:im.user/createdAt`
+  - `:im.user/profile`
+    - Required attributes for `:im.user/profile`
+      - `:im.userProfile/firstName`
+      - `:im.userProfile/lastName`
+    - Optional attributes - only include the key if it has a non-empty
+      value
+      - `:im.userProfile/displayName`
+      - `:im.userProfile/bio`
+      - `:im.userProfile/publicData`
+      - `:im.userProfile/protectedData`
+      - `:im.userProfile/privateData`
+      - `:im.userProfile/metadata`
+- Optional attributes - only include the key if it has a non-empty value
+  - `:im.user/passwordHash`
+
+Supported references
+
+- `#im.userProfile/avatar` -> `#im.image`
+
+Example syntax
+
+```
+[[:im.user/id :user/john]
+  #:im.user{:createdAt #inst "2018-04-17T06:55:04.291-00:00"
+            :primaryEmail {:im.email/address "foo@sharetribe.com"
+                          :im.email/verified true}
+            :profile {:im.userProfile/firstName "John"
+                      :im.userProfile/lastName "Doe"
+                      :im.userProfile/displayName "John D"
+                      :im.userProfile/bio "He's just a poor boy from a poor family.\nSpare him his life from this monstrosity."
+                      :im.userProfile/publicData { :premiumAccount true }
+                      :im.userProfile/avatar #im/ref :avatar/john}
+            :role [:user.role/customer :user.role/provider]}]
+```
+
+#### Image
+
+Image urls need to be public and properly encoded URLs, so that we can
+access the imported images while importing the data.
+
+- Required attributes
+  - `:im.image/url`
+- Optional attributes - only include the key if it has a non-empty value
+  - `:im.image/listing`
+  - `:im.image/sortOrder`
+
+Supported references
+
+- `#im.image/listing` -> `#im.listing`
+
+Note that for profile images, the reference is added in [user](#user)
+with `#im.userProfile/avatar` -> `#im.image`
+
+Example syntax
+
+```
+[[:im.image/id #uuid "58afd8e1-e336-4ca4-a1e7-ff1d91856a6c"]
+  #:im.image{:url "https://asset-url.someservice.com/path/to/img1.jpg"
+            :sortOrder 1
+            :listing #im/ref :listing/rock-sauna}]
+```
+
+#### Stripe account
+
+- Required attributes
+  - `:im.stripeAccount/stripeAccountId`
+  - `:im.stripeAccount/user` TODO: how about chargesEnabled and
```

Comment:
Are `chargesEnabled` and `payoutsEnabled` optional or required? They're not references and they're not mentioned in the Intermediary spec

Reply:
IIRC, the Stripe integration in FTW fetches this data and it can be accessed that way.